magefile.go

```diff
@@ -1254,6 +1257,7 @@ func (d Dev) init() error {
 			dev.WithWaitOptions([]dev.WaitOption{
 				dev.WithTimeout(2 * time.Minute),
 			}),
+			dev.WithSchemeBuilder(runtime.SchemeBuilder{operatorsv1alpha1.AddToScheme, aoapisv1alpha1.AddToScheme}),
```

Comment:
I don't think `operatorsv1alpha1.AddToScheme` is required here.

Reply:
ahh I picked it up from the work of monitoring stack PR where it was actually required haha.  (for creating OLM resources of Observability Operator via the kube client - hence, the client would be required to be backed by a scheme aware of operatorsv1alpha1 APIs)

Will clean it up right now and bring it back as a part of the MonitoringStack PR :)